frontend/src/pages/org/crawls-list.ts

```diff
@@ -209,13 +212,19 @@ export class CrawlsList extends LiteElement {
             ${when(
               this.isCrawler,
               () => html`
-                <sl-button
-                  size="small"
-                  @click=${() => (this.isUploadingArchive = true)}
+                <sl-tooltip
+                  content=${msg("Org Storage Full")}
+                  ?disabled=${this.orgStorageQuotaReached === false}
```

Comment:
It seems like this has the opposite effect of disabling the button when the storage quota is not reached.
```suggestion
                  ?disabled=${this.orgStorageQuotaReached}
```

Reply:
Gotcha!